chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,869 @@
+    text = new_text;
+  }
+
+  // Callback methods for markdownify
+  markdownify_sortByLength(a, b) {
+    // Go by order of largest to smallest
+    return b.length - a.length;
+  }
+
+  markdownify_onSortEach(tooProcess, unique_placeholder_k, count, regexp_k, body, replacer_dict, value) {
+    const replace = tooProcess[value];
+    const swap = `${unique_placeholder_k}${(count++).toString()}`;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)